db/migrate/20220921124309_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :AuthorId, null: false, foreign_key: true
+    add_reference :comments, :post, null: false, foreign_key: true
```

Comment:
- [ ] Your migrations are returning an error, kindly take a step back and check the associations between your migrations
![image](https://user-images.githubusercontent.com/62705947/191527167-c41f9650-c870-4cfe-9b98-5f512e915ee9.png)